### DISCUSSION TITLE
Installing siegfried in Google Colab

### DISCUSSION THREAD

**Discussion by sonoet** [richardlehane/siegfried#241] [Q&A] [answered]
Hi!  This might be an odd question, but I'm wondering if anyone has ever tried installing siegfried in Google Colab.  I'm teaching an online class on digital preservation and would like the students to get some hands on experience with a few tools without having to rely on everyone installing things on their machine.  I have a few things set up in Colab for them to try, but I'd really like to add a few tasks using Brunnhilde. I'm able to install Brunnhilde, but I'm having trouble installing siegfried.  I was wondering if anyone has tried this and knows how to do it.  Thank you!

**Comment by ross-spencer** [comment, marked as answer]
What a cool use-case @sonoet

You can try the following notebook commands which should download Siegfried, install it and then run it over the `/tmp` directory. 

```
!wget https://github.com/richardlehane/siegfried/releases/download/v1.11.0/siegfried_1.11.0-1_amd64.deb -O sf.deb
!dpkg -i sf.deb
!sf -update
!sf /tmp/
```

**Reply by sonoet** [reply]
@ross-spencer Wow!  Thank you so much!  Everything works!  I'm using a lot of exclamation points, but that's because I'm really excited and grateful.  This is going to be so great for the students to get to experiment with these workflows online.  This is for our class next week on accessioning and ingest workflows.  I'll also plan to show them this discussion thread so they can see what a supportive community this is.  Thanks so much!
